pipelinewise/fastsync/mysql_to_snowflake.py

```diff
@@ -131,6 +131,9 @@ def sync_table(table: str, args: Namespace) -> Union[bool, str]:
             target_schema, table, snowflake_columns, primary_key, is_temporary=True
         )
 
+        # Update table primary keys to nullable
+        snowflake.update_primary_keys_nullable(target_schema, table, primary_key, is_temporary=True)
```

Comment:
fastsync recreates the table, so we can create the table with not null instead of altering it.